conftest.py

```diff
@@ -0,0 +1,9 @@
+# List of files that pytest should ignore
+
+collect_ignore = ["setup.py",
+                  "skimage/io/_plugins"]
+try:
+    import visvis
```

Comment:
Oh noes!  We import that library in https://github.com/scikit-image/scikit-image/blob/9ffd661fe8cf7fe7b7d7236e748417e3a3f5a1c3/skimage/measure/_marching_cubes_lewiner.py, but it is not in our requirements files.